.githooks/pre-commit

```diff
@@ -1,4 +1,23 @@
 #!/usr/bin/env bash
 
-cargo make check-rs-format && \
-cargo make check-ts-format
+# Run check-rs-format if Rust source files were modified.
+git status | egrep "*.rs"
```

Comment:
:wrench: There is actually a nice command for listing only the filenames of all staged files: `git diff --name-only --cached`

Source: http://stackoverflow.com/a/33610683

This way, we even get a feature for free with this change: Checking only the files that you actually want to commit :)

Reply:
I like it, implemented as suggested.  Thanks.

Reply:
I use a loop and run with and without `--cached` though for those of use who commit without staging.